client/stylesheets/_toCat.scss

```diff
@@ -4813,6 +4810,71 @@ On click the images will be displayed at normal size to complete the effect
 }
 
 
+/*Options DropDown*/
+
+.optionsClick {
+	margin:100px auto;
```

Comment:
Line should be indented with spaces, not tabs<br>Colon after property should be followed by one space